yeezyCopper.js

```diff
@@ -60,69 +64,153 @@ function pay () {
     }
   });
 
-  $('input#dwfrm_payment_creditCard_cvn').val(function () {
+  $('input#dwfrm_payment_creditCard_cvn').val(function() {
     return config.security_code;
   });
 
-  setTimeout(function () {
+  setTimeout(function() {
     placeOrder();
   }, 300);
 }
+
+function _x(STR_XPATH) {
+  var xresult = document.evaluate(STR_XPATH, document, null, XPathResult.ANY_TYPE, null);
+  var xnodes = [];
+  var xres;
+  while (xres = xresult.iterateNext()) {
+    xnodes.push(xres);
+  }
+
+  return xnodes;
 }
 
-function addNewShipping() {
-  $('.textinput.firstname').val(function () {
-    return config.first_name;
-  });
+function sleep(milliseconds) {
```

Comment:
I setup the sleep function, just because I thought it would look nicer for the `checkovernightShip` function. Should I change it over to a `setTimeout()`? I initially put in the sleep function so it would be easier for me to program the `checkovernightShip` function; I agree it would be helpful to condense the two functions, and use the `setTimeout()` but does it change aesthetics -- do the aesthetics even matter?